src/services/refactors/convertStringOrTemplateLiteral.ts

```diff
@@ -196,12 +215,17 @@ namespace ts.refactor.convertStringOrTemplateLiteral {
                     copyExpressionComments(span);
                     const nextSpan = currentNode.templateSpans[index + 1];

```

Comment:
This is somewhat of a preexisting issue, but its better to do a bounds check (i.e. `index + 1 < currentNode.templateSpans.length`) rather than over-index, since an out-of-range indexed access results in a deopt in V8.

Reply:
Good to know; I further simplified it to just a boolean since the value wasn't used.